third_party/blink/renderer/modules/cobalt/h5vcc_runtime/h_5_vcc_runtime.h

```diff
@@ -77,12 +74,16 @@ class MODULES_EXPORT H5vccRuntime final
 
  private:
   void MaybeFireDeepLinkEvent(const String&);
-  void EnsureReceiverIsBound();
-  void RemoteAddListener();
+  void EnsureRemoteIsBound();
+
+  void MaybeRegisterMojoListener();
+  void MaybeUnregisterMojoListener();
+
   HeapMojoRemote<h5vcc_runtime::mojom::blink::H5vccRuntime>
       remote_h5vcc_runtime_;
   HeapMojoReceiver<h5vcc_runtime::mojom::blink::DeepLinkListener, H5vccRuntime>
-      receiver_;
+      listener_receiver_;
```

Comment:
Since we're renaming this, I think this name should be `deep_link_listener_` or `deep_link_receiver_`.